### PR TITLE
test

### PR DESCRIPTION
**Pull request by MicheleBertoli**
yuno

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha --recursive"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/unlucio/promiseKit"
+  },
+  "author": "unlucio",
+  "license": "ISC",
```
Comment (unlucio) [inline package.json]: can we have MIT? :D

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "bugs": {
+    "url": "https://github.com/unlucio/promiseKit/issues"
+  },
+  "homepage": "https://github.com/unlucio/promiseKit",
+  "devDependencies": {
+    "mocha": "^2.2.5"
+  },
+  "engine": "node 0.12.7"
```
Comment (unlucio) [inline package.json]: I'm all for "if you don't update you're  a bad person" but perhaps we should allow all the 0.12 vintage versions as well? :)

**Comment by MicheleBertoli** [comment]
done @unlucio ;)

**Comment by MicheleBertoli** [comment]
done @unlucio ;)

**Comment by unlucio** [comment]
<3

**Pull request merged**